Packs/FeedMitreAttack/IndicatorFields/incidentfield-mitremodified.json

```diff
@@ -0,0 +1,30 @@
+{
+ "id": "indicator_mitremodified",
+ "version": -1,
+ "modified": "2020-09-13T16:40:46.057661+03:00",
+ "name": "Modified (technique)",
```

Comment:
why do we have `technique` here? also lets add `MITRE` here so this wont be too generic